src/tomczyk-kamil/Fsd.Kamil.Client/Fsd.Kamil.Client.Ex1/Style.scss

```diff
@@ -0,0 +1,144 @@
+@mixin pageSize($left, $right, $top, $bottom) {
```

Comment:
bardzo fajnie że użyłeś sass, to na plus dla Ciebie. Szkoda tylko że nie podzieliłeś tego na pliki. Zrób jeżeli uważasz że warto